try {
    // Protected Code
} catch (ExceptionName ex) {
    // Catch Block
}